README.md

```diff
@@ -1,9 +1,9 @@
 # Omron MIT Elite Plus HEM-7301-ITKE7
-Python script to extract measurements from an Omron MIT Elite Plus (HEM-7301-ITKE7) blood pressure monitor with USB support (0590:0028).
+Python script to extract measurements from an Omron MIT Elite Plus (`HEM-7301-ITKE7`) blood pressure monitor with USB support (`0590:0028`). This has also been tested to work with an Omron T9P (`HEM-759P-C1`) using the `HHX-CABLE-USB1`. This cable claims to work with `HJ-720IT`, `HEM-7080IT-E` and `HEM-790IT` models, although this has not been tested yet.
 
 ***
 
-Based on [https://usb2me.wordpress.com/2013/02/06/omron-mit-elite-plus-hem-7301-itke7](https://web.archive.org/web/20200704141615/https://usb2me.wordpress.com/2013/02/06/omron-mit-elite-plus-hem-7301-itke7/)
+Based on [https://usb2me.wordpress.com/2013/02/06/omron-mit-elite-plus-hem-7301-itke7](https://web.archive.org/web/20200704141615/https://usb2me.wordpress.com/2013/02/06/omron-mit-elite-plus-hem-7301-itke7/) ([direct link](https://usb2me.wordpress.com/2013/02/06/omron-mit-elite-plus-hem-7301-itke7)).
```

Comment:
Feel free to remove the archive.org link altogether; I added it just in case the original site disappeared, but maybe it's too soon to do that.